hc/api/tests/test_admin.py

```diff
@@ -17,3 +20,4 @@ def test_it_shows_channel_list_with_pushbullet(self):
         ch.save()
 
         ### Assert for the push bullet
+        self.assertEqual(ch.kind, "pushbullet")
```

Comment:
Checking for the kind in the `ch` is not a conclusive test. The idea is that a channel is added in the lines above, so you have to check that the overall channels have the new channel with the kind `pushbullet`

Reply:
@mutebironald Lines 19 and 20 create a channel and save it. Check in the list of saved channels for one that has `pushbullet`